app/code/Magento/Customer/view/frontend/web/js/customer-data.js

```diff
@@ -99,7 +99,7 @@ define([
             parameters = _.isArray(sectionNames) && sectionNames.indexOf('*') < 0 ? {
                 sections: sectionNames.join(',')
             } : [];
```

Comment:
Looks like it should be all the time object
```suggestion
            parameters = _.isArray(sectionNames) && sectionNames.indexOf('*') < 0 ? {
                sections: sectionNames.join(',')
            } : {};
```